### PR TITLE
RDKB-44513 - Free the memory leaks from rbus gtest

### PR DESCRIPTION
**Pull request by dm097**
Reason for change: Gtest Suits listed in Jira RDKB-44513 Description were failing
Test Procedure: Build rtmessage, rbuscore and start rtrouted", Build Rbus Gtests with valgrind mentioned in Jira Description
Risks: Low
Priority: P2
Signed-off-by: Deepak <deepak_m@comcast.com>

### CODE REVIEW

File: src/rbus/rbus.c [hunk 1]
```diff
@@ -2689,8 +2689,8 @@ rbusError_t rbus_get(rbusHandle_t handle, char const* name, rbusValue_t* value)
                 errorcode = CCSPError_to_rbusError(legacyRetCode);
             }
         }
-
-        rbusMessage_Release(response);
+        if(response != NULL)//check if "response" is already Freed And Set to Null in rbus_invokeRemoteMethod()
```
Comment (karuna2git) [inline src/rbus/rbus.c:2692]: this check may not be needed as it is already freed by rbusMessage_Release() function per recent change.

**Review by karuna2git — COMMENTED** [review]
Please review the comment in rbus.c; everything else looks good to me.

File: src/rbus/rbus.c (hunk 1 above)
Comment (dm097) [reply src/rbus/rbus.c]: Removed this if(response !=NULL) as per your review comment. Changes are committed as part of commit a9e1fdaf3edf419bb45e7b3470ed54224b04aeb6.

**Review by karuna2git — COMMENTED** [review]
in rbusProvider.cpp, Line 88, the timeVal is not inited. Try memset `timeVal` to 0.

**Review by karuna2git — APPROVED** [review]

**Pull request merged**